.githooks/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+STANDARD="Drupal,DrupalPractice"
+BIN="./vendor/bin"
```

Comment:
unused variable declared. can we remove this line?

Reply:
should I remove both the variables? 

> unused variable declared. can we remove this line?

Should I remove both the variables?